lib/system_parser.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+class SystemParser
+  DfRecord = Struct.new(:unix_device, :optional_name, :size_gib, :avail_gib)
+
+  def self.get_device_mount_points_from_lsblk(json_input, device_name)
```

Comment:
nit: maybe rename to get_device_mount_points_from_lsblk_json

Reply:
done